packages/libs/react-ui/src/components/MaskedValue/MaskedValue.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { MaskedValue } from '@components/MaskedValue';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+describe('MaskedValue', () => {
+  test('renders correctly', () => {
+    const { getByTestId } = render(<MaskedValue value="TestValue1234" />);
+
+    const maskedValue = getByTestId('kda-masked-value');
+    expect(maskedValue).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByText('TestVa****1234').toBeInTheDocument();
```

its best practice to avoid directly checking `testId`. by checking `testId` we're not really testing the implementation or user expectations of the component.

we could also add one more test case to check when we toggle the visibility it's showing the Unmasked values.